modules/persistent-storage-csi-vsphere-encryption-datastore-url.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// persistent-storage-csi-vsphere.adoc
+//
+
+:content-type: PROCEDURE
+[id="persistent-storage-csi-vsphere-encryption-datastore-url_{context}"]
+= Using datastore URL
+
+.Procedure
+
+To encrypt using the datastore URL:
+
+. Find out the name of the default storage policy in your datastore that supports encryption. 
++
+This is same policy that was used for encrypting your VMs. 
+
+. Create a storage class that uses this storage policy:
```

Comment:
Suggestion: I think this could use more explanation, possibly including callouts in the YAML that point to the required fields.